### PR TITLE
Assignment 4

### PR DESCRIPTION
**Pull request by JSLWilfizz**
I made a mistake, so I had to revert one of the commits.

### CODE REVIEW

File: Assignment4/JSLWilfizz-999927/task07.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  }
+  ''', initNs={"ns": ns}
+                  )
+for r in g.query(q1):
+    print(r.sub)
+print("--------------------------------")
+#With RDFlib
+
+for s, p, o in g.triples((None, RDFS.subClassOf, ns.LivingThing)):
+    print(s)
```
Comment (anaigmo) [inline Assignment4/JSLWilfizz-999927/task07.py:44]: missing recursive sublcass of relationships

File: Assignment4/JSLWilfizz-999927/task07.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+for r in g.query(q1):
+    print(r.persona)
+
+q2 = prepareQuery('''
+  SELECT  ?person WHERE {
+    ?sub rdfs:subClassOf ns:Person.
+    ?person a ?sub.
+  }
+  ''', initNs={"ns": ns}
+                  )
```
Comment (anaigmo) [inline Assignment4/JSLWilfizz-999927/task07.py:71]: missing again recursive subclassof relationships, and the queries can be merge into one

File: Assignment4/JSLWilfizz-999927/task07.py [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+print("--------------------------------")
+#With RDFlib
+
+
+for s, p, o in g.triples((None, RDFS.subClassOf, ns.LivingThing)):
+    for s1, p1, o1 in g.triples((None, None, s)):
+      print(s1)
+      for s2, p2, o2 in g.triples((s1, None, None)):
+       print(s2,p2,o2)
```
Comment (anaigmo) [inline Assignment4/JSLWilfizz-999927/task07.py:114]: this is subclass of living thing, not what is asked in the exercise :)

File: Assignment4/JSLWilfizz-999927/task07.py [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+dicc = defaultdict(int)
+
+for name, p, o in g.triples((None, FOAF.knows, None)):
+  dicc[name] += 1
+for name in dicc.keys():
+  if dicc[name] >= 2:
+    for s, p, o in g.triples((name, VCARD.FN, None)):
+        print(f"{o} know {dicc[name]} entities")
+# Visualize the results
```
Comment (anaigmo) [inline Assignment4/JSLWilfizz-999927/task07.py:142]: missing sparql alternatives

**Comment by JSLWilfizz** [comment]
deleting old pull request for assignment 4

**Pull request closed without merging**